.changeset/chilled-teachers-bathe.md

```diff
@@ -0,0 +1,56 @@
+---
+'@as-integrations/aws-lambda': major
+---
+
+## Why Change?
+
+In the interest of supporting more event types and allowing user-extensibility, the event parsing has been rearchitected. The goal with v2.0 is to allow customizability at each step in the event pipeline, leading to a higher level of Lambda event coverage (including 100% custom event requests).
+
+## What changed?
+
+The second parameter introduces a handler that controls parsing and output generation based on the event type you are consuming. Not only did are there the 3 main event types we alreadys support, but there is a function for creating your own event parsers too in the case the pre-defined ones are not sufficient.
```

Comment:
```suggestion
The second parameter introduces a handler that controls parsing and output generation based on the event type you are consuming. We support 3 event types out-of-the-box: APIGatewayProxyV1/V2 and ALB. Additionally, there is a function for creating your own event parsers in case the pre-defined ones are not sufficient.
```